README.md

```diff
@@ -29,32 +29,30 @@ We need your help to make ODK Central Frontend as useful as possible! Please see
+- Install Node.js 20.
+- Install dependencies by running `npm install`.
+- Install NGINX.
+- Set up [ODK Central Backend](https://github.com/getodk/central-backend).
+  - You will need to create a user using an ODK Central Backend command line script.
+  - You will probably also want to promote that user to a sitewide administrator.
 
 ## Running in development
 
 Follow these instructions to run ODK Central Frontend in development. For deploying to production, see the next section.
 
 First, run ODK Central Backend.
 
-Next, build ODK Central Frontend files for development by running `npm run dev`.  The files will be outputted to `dist/`.  As you update the source code, the files will be automatically rebuilt.  `npm run dev` will also start NGINX, which will serve the files.
-
-NGINX effectively places ODK Central Frontend and ODK Central Backend at the same origin, avoiding cross-origin requests.
+Next, build ODK Central Frontend files for development by running `npm run dev`.  The files will be outputted to `dist/`.  As you update the source code, the files will be automatically rebuilt.  `npm run dev` will also start NGINX, which will serve the files. NGINX effectively places ODK Central Frontend and ODK Central Backend at the same origin, avoiding cross-origin requests.
 
 ODK Central Frontend will be available on port 8989.
 
 ODK Central Frontend communicates with ODK Central Backend in part using a session cookie. The cookie is `Secure`, but will be sent over HTTP on localhost. ODK Central Frontend also interacts with data collection clients and with services:
 
 - To upload an XLSForm, run [pyxform-http](https://github.com/getodk/pyxform-http). ODK Central Frontend communicates with pyxform-http through ODK Central Backend.
 - You can use ODK Collect to scan an app user QR code, download a form, and submit data. One option to do so is to use [`ngrok`](https://ngrok.com/download). ODK Central Frontend is available on port 8989, so you can run `ngrok http 8989` to expose a temporary HTTPS URL that you can use. Within ODK Central Backend, you will also need to set the `default.env.domain` property in [`config/default.json`](https://github.com/getodk/central-backend/blob/master/config/default.json) to the HTTPS URL, then restart ODK Central Backend if it is already running.
-- Enketo is a web form engine used to show form previews and allow for web-based data entry. Please see our [instructions](docs/enketo.md) for optionally setting up an Enketo server for use in _development_ (it is already included in the production ODK Central stack).
-
+- Enketo is a Web Form engine used to show Form previews and allow for web-based data entry. Please see our [instructions](/docs/enketo.md) for optionally setting up an Enketo server for use in _development_ (it is already included in the production ODK Central stack).
```

Comment:
In other places, we usually use absolute paths to other files, so I made this path absolute as well.